shell/models/workload.service.js

```diff
@@ -0,0 +1,314 @@
+
+import { findBy } from '@shell/utils/array';
```

Comment:
Why is there a new workload service model?

Reply:
There is overlap between `/Users/shavindra/Documents/dev/dashboard/shell/models/workload.js` and `/Users/shavindra/Documents/dev/dashboard/shell/models/pod.js`

So these were moved to their own service.